### PR TITLE
feat(BDM access control) new BDM access control page

### PR DESCRIPTION
**Pull request by abirembaut**
- Add documentation for the BDM access control page
- Fix BDM import page path

Covers [BS-17264](https://bonitasoft.atlassian.net/browse/BS-17264)

### CODE REVIEW

File: md/bdm-management-in-bonita-bpm-portal.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The Business objects must match the structure used by the deployed processes. Ma
 **:fa-info-circle: Note:** The deployment of a Business Data Model requires pausing the service during the operation, so that the update can be performed without affecting ongoing processes. 
 :::
 
-5. When the service is paused, go to the **Business Data Model** menu.
+5. When the service is paused, go to the **Business Data Model** menu (and, for Performance and Efficiency editions, choose **Definition**).
 6. A page called **Import and activate a new Business Data Model** is displayed.
 7. Choose the file containing the Business Data Model exported from the Bonita Studio, and click on _**Activate**_.
```
Comment (educhastenier) [inline md/bdm-management-in-bonita-bpm-portal.md]: from ~the~ Bonita Studio

File: md/bdm-management-in-bonita-bpm-portal.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The Business objects must match the structure used by the deployed processes. Ma
 **:fa-info-circle: Note:** The deployment of a Business Data Model requires pausing the service during the operation, so that the update can be performed without affecting ongoing processes. 
 :::
 
-5. When the service is paused, go to the **Business Data Model** menu.
+5. When the service is paused, go to the **Business Data Model** menu (and, for Performance and Efficiency editions, choose **Definition**).
```
Comment (educhastenier) [inline md/bdm-management-in-bonita-bpm-portal.md]: for Efficiency and Performance (change order)

File: md/bdm-management-in-bonita-bpm-portal.md [hunk 3]
```diff
@@ -35,4 +35,25 @@ in the business database, so as to store business objects appropriately when mod
 :::
 
 It is recommended to perform a backup of the business database before deploying a business data model.
-(If the deployment fails, check the engine logs to analyse the problem.)
+(If the deployment fails, check the engine logs to analyse the problem.)  
+
+::: info
+**Note:** For Performance and Efficiency editions only.
```
Comment (educhastenier) [inline md/bdm-management-in-bonita-bpm-portal.md]: for Efficiency and Performance (change order)

File: md/bdm-management-in-bonita-bpm-portal.md [hunk 4]
```diff
@@ -35,4 +35,25 @@ in the business database, so as to store business objects appropriately when mod
...
+2. In the Bonita Portal, log in as the technical user (default login 'install', default password 'install') or as a user with the administrator profile.
+3. As the technical user, go to the **Business Data Model** menu and choose **Access Control**. As an administrator, go to the **BDM** menu.
+4. A page with a section **Import a BDM access control file** is displayed.
+5. Choose the file containing the Business Data Model exported from the Bonita Studio, and click on _**Import**_.
+6. A successful import message will be displayed.
+
+Please note that a delay is required after the import for the engine to process the access control and for the filtering to be effective.
+
+::: info
+**:fa-info-circle: Note:** Contrary to the BDM definition, the deployment of a Business Data Model access control file doesn't require pausing the service during the operation.
```
Comment (educhastenier) [inline md/bdm-management-in-bonita-bpm-portal.md]: doesn't require pausing the BPM services

File: md/bdm-management-in-bonita-bpm-portal.md [hunk 5]
```diff
@@ -35,4 +35,25 @@ in the business database, so as to store business objects appropriately when mod
...
+:::
+
+It is possible to define Business Data Model access control rules in Bonita Studio and import them in Bonita Portal.  
+The Access control rules must match the Business Data model.
+
+1. To import the BDM access control rules created in Bonita Studio, first [export the Access Control file](bdm-access-control.md) from Bonita Studio where it was designed.
+2. In the Bonita Portal, log in as the technical user (default login 'install', default password 'install') or as a user with the administrator profile.
+3. As the technical user, go to the **Business Data Model** menu and choose **Access Control**. As an administrator, go to the **BDM** menu.
+4. A page with a section **Import a BDM access control file** is displayed.
+5. Choose the file containing the Business Data Model exported from the Bonita Studio, and click on _**Import**_.
```
Comment (educhastenier) [inline md/bdm-management-in-bonita-bpm-portal.md]: Choose the file containing the Business Data Model +access control definition+ exported from ~the~ Bonita Studio.

File: md/define-and-deploy-the-bdm.md [hunk 6]
```diff
@@ -213,7 +213,7 @@ Follow these steps:
 5. Log in to Bonita Portal as the technical user.
 6. Go to the **BPM Services** page.
 7. Click **Pause**.
-8. When the service is paused, go to the **Resources** menu and choose **Business Data Model**.
+8. When the service is paused, go to the **Business Data Model** menu (and, for Performance and Efficiency editions, choose **Definition**).
```
Comment (educhastenier) [inline md/define-and-deploy-the-bdm.md]: for Efficiency and Performance (change order)

**Comment by abirembaut** [comment]
feedback integrated

**Comment by educhastenier** [comment]
Looks good to me :+1:

**Comment by educhastenier** [comment]
According to https://bonitasoft.atlassian.net/browse/BS-17264, there should be a new doc page in taxonomy?

**Comment by abirembaut** [comment]
@educhastenier : that's in another PR : https://github.com/bonitasoft/bonita-doc/pull/387

**Pull request merged**